### DISCUSSION TITLE
SvelteKit `formaction`'s - Bug or Issue?

### DISCUSSION THREAD

**Discussion by rogadev** [sveltejs/kit#8862] [Q&A]
Am I introducing a bug or is this an issue?

When I put a button with social auth `formaction="?/login&provider=google"` in the same form as a button with a `formaction` to submit without the provider param, the form tries to do the default action. Even if I have 2 forms, one that wraps my social auth and one that wraps my email/password auth, it still tries to use the default action.

What's worse is that can clearly see that it is using the formaction that I intended, because it is trying to route to `<hostname>/?login&provider=google`.

Did I find an issue with SvelteKit? Why is it trying to use the default action when no default action is specified?

Here's a gist you can have a look at: [https://gist.github.com/rogadev/5ef318c54bc81c95c0909ae4165d9e52](https://gist.github.com/rogadev/5ef318c54bc81c95c0909ae4165d9e52)

**Comment by teemingc** [comment]
I tried reproducing the issue but couldn't figure it out what went wrong in the gist you shared.

Here's a working example using the form you shared
https://stackblitz.com/edit/sveltejs-kit-template-default-enyxz3?file=src%2Froutes%2F%2Bpage.server.js,src%2Froutes%2F%2Bpage.svelte&terminal=dev
